catapult-sdk/src/plugins/aggregate.js

```diff
@@ -89,6 +89,7 @@ const requireCodecs = txCodecs => {
 const aggregatePlugin = {
 	registerSchema: builder => {
 		const aggregateSchema = {
+			transactionHash: ModelType.binary,
```

Comment:
this has been implemented prematurely, it's a bugfix from a field that is not in core yet but has to be added